### ISSUE TITLE
This driver does not work in wayland

### ISSUE THREAD

**Issue by oneXfive** [teddywlq/smifb2#15]
I used Ubuntu22.04 OS and found that it did not work in wanland. But the xorg desktop works fine.

**Comment by teddywlq** [comment]
Yes. I'm changing the driver from VRAM Helper to SHMEM Helper. That will let it work again with wayland

**Comment by marafa-sugarcrm** [comment]
has this been resolved yet?
